### PR TITLE
adding example template for MonoLightZprime UL 2017 for single mass point

### PR DESCRIPTION
**Pull request by abhi-mal**
Requesting for central production for Ultralegacy for 2016(preVFP and post VFP),2017,2018

### CODE REVIEW

**Comment by menglu21** [comment]
please update your local git repo firstly before create your PR since you can see this pr has several thousands files changed.

**Comment by abhi-mal** [comment]
Hi,
I removed the files that were present in the gridpack by default and retainted only the cards and the model files. Please let me know if there is something else I should do.

**Comment by menglu21** [comment]
please move the cards to directory "MadGraph5_aMCatNLO/cards/production/2017/13TeV" for UL, and send the compressed model file to us, then we will upload it to the central place where it can be called directly from the gridpack generation script.

**Comment by abhi-mal** [comment]
Hi,
I moved the files as suggested. The compressed model file is available here[1]. Please let me know if you need anything else. 


[1] https://pages.hep.wisc.edu/~varuns/monoZprime/model/DMEFT2ZpNewV2_UFO.zip

Thanks,
Abhi

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#*                                                          *
+#************************************************************
+set default_unset_couplings 99
+set loop_color_flows False
+set max_npoint_for_channel 0
+set loop_optimized_output True
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_proc_card.dat:30]: please rm the banner

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+# WARNING: Do not use for interference type of computation           *
+#*********************************************************************
+  True	= use_syst ! Enable systematics studies
+#
+  systematics	= systematics_program ! none, systematics [python], SysCalc [depreceted, C++]
+['--mur=0.5,1,2', '--muf=0.5,1,2', '--pdf=errorset'] = systematics_arguments ! see: https://cp3.irmp.ucl.ac.be/projects/madgraph/wiki/Systematics#Systematicspythonmodule
+# Syscalc is deprecated but to see the associate options type'update syscalc'#********************************************************************* 
+#  Additional hidden parameters
+#*********************************************************************
+  ['--mur=0.5,1,2', '--muf=0.5,1,2', '--pdf=errorset']	= systematics_arguments # Choose the argment to pass to the systematics command. like --mur=0.25,1,4. Look at the help of the systematics function for more details.
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_run_card.dat:284]: these lines can be removed

**Review by menglu21 — COMMENTED** [review]
the model has been moved to central place, please see other comments inline

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_run_card.dat (hunk 2 above)
Comment (abhi-mal) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_run_card.dat]: Hi, 
I made the suggested changes to the example cards present here. 

Do you want me to remake the gridpacks for all mass parameters with these changes(i.e. removing banner from process card and removing these extra lines from run card) or can we use the previously generated gridpacks?

Comment (menglu21) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/MonoLightZprime/Mx1_Mv1000_cards/DM_MonoZPrime_V_Mx1_Mv1000_gDM1gSM0p25_Zprime1p0_run_card.dat]: hi, it should be ok, in https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/Utilities/gridpack_helpers.sh#L63-L64 we disable the calculation automatically.

**Comment by abhi-mal** [comment]
Hi,
Thanks for this. 

Could you please upload the sister model[1] also to the central repository. The models are identical except the first one is a Simplified Dark matter model with Vector mediator and this one is with an axial mediator. The gridpacks are made using the same code too. 

[1] https://pages.hep.wisc.edu/~mallampalli/Analysis/MonoZprime/UL/axialvector_model/DMEFT2ZpNewV2_UFO.zip

**Comment by abhi-mal** [comment]
Hi,
I also noticed that the model zip filenames are same in both the earlier vector mediator model and the new axial vector model. 

Please let me know if that might cause issues and I will change the names.

**Pull request merged**